lib/python/TI/GPIO/gpio_pin_data.py

```diff
@@ -252,6 +298,9 @@ def warn_if_not_carrier_board(*carrier_boards):
         model = AM68_SK
     elif matches(compats_am69sk):
         model = AM69_SK
+    elif matches(compats_am62ask):
+        model = AM62A_SK
+
```

Comment:
Why this extra empty line
is this python black thing?

Reply:
Yes, Rahul it is formatted in PEP 8 compliant supported by black